cmd/flowrunner/testdata/flows/brochure_test.json

```diff
@@ -406,10 +407,6 @@
                     },
                     "triggered_on": "2000-01-01T00:00:00Z",
                     "type": "manual"
-                },
```

Comment:
there was a bug previously where waits wouldn't be cleared so runs would end with waits